packages/graphql-yoga/src/plugins/resultProcessor/stringify.ts

```diff
@@ -5,7 +5,13 @@ import type { MaybeArray } from '../../types.js'
 // JSON stringifier that adjusts the result extensions while serialising
 export function jsonStringifyResult(result: MaybeArray<ExecutionResult>) {
   return JSON.stringify(result, (key, value) => {
```

Comment:
Should we do something like

```
`{
  "data":  ${JSON.stringify(result.data)},
  "errors": ${JSON.stringify(result.errors, custom logic)},
  "extensions": ${JSON.stringify(result.error, custom logic)}
}`
```

To avoid special behavior in the "data" property JSON tree?

Reply:
Good point, let me see what I can do.

Reply:
https://github.com/dotansimha/graphql-yoga/pull/2331/commits/46cbcd1575ee5ac9cc0ab65b0331a788b3cb1c8f